templates/artist/artist_list.html

```diff
@@ -95,9 +96,11 @@
         <hr>
     </div>
     <p>Are you an artist? Interested in being on PerDiem?</p>
-    <a href="{% url 'artist_application' %}" class="button">Apply Here</a>
-    <div class="break">
-        <hr>
+    <div class="apply-button">
+        <a href="{% url 'artist_application' %}" class="button">Apply Here</a>
+        <div class="break">
+            <hr>
+    </div>
```

Comment:
This `</div>` on line 103 closes the `<div>` on line 101, so it should be indented to match that one.